website/content/docs/agent/wal-logstore/index.mdx

```diff
@@ -0,0 +1,42 @@
+
+To mitigate risks associated with sudden bursts of log data, Consul prevents too many logs from accumulating in the LogStore. Significantly larger BoltDB files are generally slower to process because the file is structured as a tree. As a result, Consul must iterate through the log for each write. But the larger the file, the more likely it is to have a large freelist or suddenly form one after a burst of writes. For this reason, the many of Consul's default options asssociated with snapshots, truncating logs, and keeping the log history have been aggressively set toward keeping BoltDT small rather than uisng disk IO more efficiently. 
+
+Other reliability issues, such as [raft replication capacity issues](/consul/docs/agent/telemetry#raft-replication-capacity-issues), are also related to balancing the size of the BoltDB log store and length of snapshots. 
+
+Although some users may not experience a significant difference in performance, the WAL backend avoids performance concerns associated with BoltDB. When directly measured, WAL is more performant than BoltDB because it solves a simpler storage problem than BoltDB was designed to solve. For example, WAL can commit a single log entry with one `fsync` instead
+of two and tends to write two to three fewer bytes to the disk to commit the `fsync`. The primary benefit is that retaining more logs does not affect write performance. As a result, strategies for reducing disk IO with slower snapshots or for keeping logs to permit slower followers to catch up with cluster state are all possible.
+
+## WAL quality assurance
+
+The WAL backend has been tested thoroughly during development:
+
+- Every component in the WAL, such as [metadata management](https://github.com/hashicorp/raft-wal/blob/main/types/meta.go), [log file encoding](https://github.com/hashicorp/raft-wal/blob/main/types/segment.go) to actual [file-system interaction](https://github.com/hashicorp/raft-wal/blob/main/types/vfs.go) were abstracted so unit tests can simulate difficult-to-reproduce disk failures.
+ 
+- We used the [application-level intelligent crash explorer (ALICE)](https://github.com/hashicorp/raft-wal/blob/main/alice/README.md) to exhaustively simulate thousands of possible crash failure scenarios. WAL correctly recovered from all scenarios.
+
+- We ran hundreds of tests in a performance testing cluster with checksum verification enabled and did not detect data loss or corruption. We will continue testing before making WAL the default backend.
+
+We are aware of how complex and critical disk-persistence is for your data.
+
+Our goal is that many users at all degrees of scale try WAL in their environments after upgrading to 1.15 or later and report success or failure so that we can confidently replace BoltDB before as the default for new clusters.
```

Comment:
```suggestion
We hope that many users at different scales will try WAL in their environments after upgrading to 1.15 or later and report success or failure so that we can confidently replace BoltDB as the default for new clusters in a future release.
```